### DISCUSSION TITLE
[Enhancement] [input-number] Remove trailing zeros in el-input-number display value

### DISCUSSION THREAD

**Discussion by li936393** [element-plus/element-plus#18192] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `input-number`

## Description

When using the el-input-number component, even though the precision=4 setting allows for values like 0.1234 to be displayed correctly, entering a value of 0 results in a display of 0.0000. It would be beneficial to introduce a new attribute that allows users to configure whether trailing zeros are displayed in the value.

### Additional comments

_(empty)_

**Comment by tuskermanshu** [comment]
我认为可以像 `el-input` 一样 提供一个 `format` 属性，允许用户提供一个自定义的格式化函数

**Reply by li936393** [reply]
format也可以的，当precision大一点，比如6位 8位的时候，0 1的场景下会出现0.00000000、1.00000000
